### ISSUE TITLE
[Request] Add Internet Download Manager

### ISSUE THREAD

**Issue by susanonec** [ScoopInstaller/Extras#7942]
### Information

**Name:** Internet Download Manager

**Description:** Internet Download Manager (IDM) is a tool to increase download speeds by up to 5 times, resume and schedule downloads.

**Homepage:** https://www.internetdownloadmanager.com/

**Download link(s):** https://www.internetdownloadmanager.com/download.html

**Some indication of popularity/repute:** I think it was used by many people(as it says it has over 2 Mio. users on its website), just don't know why it wasn't mentioned in this bucket.

**Comment by niheaven** [comment]
The installer of IDM cannot be expanded by known tools, and it will install some system drivers which is hard to maintain.

If someone else finds some method, welcome to submit a PR.

**Comment by niheaven** [comment]
Yes, it is. The only thing you should notice is the license field.

**Comment by Zliced13** [comment]
Ok, here are some things I found about the installer.

1. The installer file has a silent switch (/skipdlgs), there is no way to extract the files inside the installer file.
2. The uninstaller has no silent switches, and it forcefully restarts your computer at the end of the uninstallation process.

I don't think this program can be implemented in Scoop, as the one of the core parts of Scoop is to have a GUI free installation/uninstallation process.

The OP should stick with the package implemented in winget: 
`winget install Tonec.InternetDownloadManager`
